### PR TITLE
fix(markdown): Fix typos in designs documentation

### PR DESCRIPTION
**Pull request by BenJamesBen**
I went through all the designs documentation with a spellchecker and fixed all the typos I found.

- I tried to skip terms that were ambiguous about their spelling correctness, like sewing terminology and other words that might not have an agreed-upon correct spelling.
- I tried to skip grammar, punctuation, and whitespace issues.

If I made any mistakes or changed words that were actually correct or were spelling variants, please let me know and I'll change them back.

For all the affected designs, I will add the pattern designer as requested reviewers for this PR. This is so they are notified of the prospective changes to their designs and can review them.

### CODE REVIEW

File: markdown/org/docs/designs/bee/instructions/en.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ There are also notches on either side of the band piece's midpoint to help give
 
 <Note>
 
-You may prefer to wrap the one cup over the other at the midpoint. To do this place the sides of the cups at the otuer notches and lay their fronts pass the midpoint with the left on top.
+You may prefer to wrap the one cup over the other at the midpoint. To do this place the sides of the cups at the other notches and lay their fronts pass the midpoint with the left on top.
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/designs/bee/instructions/en.md:88]:
```suggestion
You may prefer to wrap the one cup over the other at the midpoint. To do this place the sides of the cups at the outer notches and lay their fronts pass the midpoint with the left on top.
```

File: markdown/org/docs/designs/hi/options/hungry/en.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: "Hungry"
 ---
 
-This setting determins how long it has been since your Hi has had
+This setting determine how long it has been since your Hi has had
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/designs/hi/options/hungry/en.md:5]:
```suggestion
This setting determines how long it has been since your Hi has had
```

**Review by bobgeorgethe3rd — CHANGES_REQUESTED** [review]

File: markdown/org/docs/designs/charlie/options/beltloops/en.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: "Belt loops"
 ---
 
-Constrol how many beltloops we'll generate.
+Control how many beltloops we'll generate.
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/designs/charlie/options/beltloops/en.md:5]:
```suggestion
Controls how many beltloops we'll generate.
```

File: markdown/org/docs/designs/simone/options/seperatebuttonplacket/en.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 title: "Seperate button placket"
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/designs/simone/options/seperatebuttonplacket/en.md:2]:
```suggestion
title: "Separate button placket"
```

File: markdown/org/docs/designs/simone/options/seperatebuttonholeplacket/en.md [hunk 5]
```diff
@@ -2,9 +2,9 @@
 title: "Seperate buttonhole placket"
```
Comment (bobgeorgethe3rd) [inline markdown/org/docs/designs/simone/options/seperatebuttonholeplacket/en.md:2]:
```suggestion
title: "Separate buttonhole placket"
```

**Review by joostdecock — APPROVED** [review]
You know what, I was wrong.
Turns out we do have a documentation team. It's one guy named Ben 😉

**Pull request merged**